src/validators/ivaDefectsByManualValidator.ts

```diff
@@ -0,0 +1,13 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
```

Comment:
Same for this one, I wonder if it's worth having an IvaDefectsValidator class, which includes your validateIvaDefectManualQuery method, as there will likely be more validation to come.